lib/checkPullRequestLabels.js

```diff
@@ -36,7 +35,7 @@ const {
 } = require('../userWhitelist.json');
 const DEFAULT_CHANGELOG_LABEL = 'REVIEWERS: Please add changelog label';
 const PR_LABELS = ['dependencies', 'stale', DEFAULT_CHANGELOG_LABEL];
-
+const utilityModule = require('./utils');
```

Comment:
Why you change the way we do the imports?

Reply:
This is required to spy on the sleep function in the frontend tests.

Reply:
Hmm, weird, doesn't it suffice to just import it in the spec file this way?

Reply:
I think it's required to be this way for the sleep function to be mocked appropriately.